rpc/src/cache_block_meta_service.rs

```diff
@@ -60,9 +64,42 @@ impl CacheBlockMetaService {
         Self { thread_hdl }
     }
 
-    fn cache_block_meta(bank: &Bank, blockstore: &Blockstore) -> Result<(), BlockstoreError> {
-        blockstore.cache_block_time(bank.slot(), bank.clock().unix_timestamp)?;
-        blockstore.cache_block_height(bank.slot(), bank.block_height())
+    fn cache_block_meta(
```

Comment:
The new contents of this function is the most interesting part of this PR and probably deserves the most attention

Reply:
Looks correct to me.